basic-algorithms.js

```diff
@@ -1,7 +1,76 @@
 // Names and Input
+var hacker1 = "Shrek";
 
+console.log(`The driver's name is ${hacker1}`);
 
-//Conditionals
+var hacker2 = window.prompt("What's the navigator's name?");
 
+console.log(`The navigator's name is ${hacker2}`);
+
+
+// Conditionals
+
+if (hacker1.length > hacker2.length) {
+    console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+} else if (hacker1.length < hacker2.length) {
+    console.log(`Yo, navigator got the longest name, it has ${hacker2.length} characters`);
+// (hacker1.length === hacker2.length)
+} else {
+    console.log(`wow, you both got equally long names, ${hacker1.length} characters!!`);
+}
+
+
+// Loops
+var printDriver;
```

Comment:
Here you need to assign the printDriver to an empty string,
var printDriver = ""